packages/application-shell/src/components/login/login.js

```diff
@@ -170,70 +171,78 @@ export class Login extends React.PureComponent {
           return this.handleSubmit(values, addHandlers);
         }}
         render={formikProps => (
-          <LoginBox>
```

Comment:
Can we try to keep this?

I'm not sure if you know but we also have a second login screen for SSO (`/login/sso`)

Reply:
Do we really need it tho? I realized I didn't need it because it was just an old version of the Card with overridden styles, but the new designs are totally compatible with just using the `Card` component as-is.